src/Altinn.Platform/Altinn.Platform.Events/Functions/Models/Subscription.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Text.Json;
+
+namespace Altinn.Platform.Events.Models
+{
+    /// <summary>
+    /// Class that describes a events subscriptions
+    /// </summary>
+    public class Subscription
+    {
+        /// <summary>
+        /// Endpoint to receive matching events
+        /// </summary>
+        public Uri EndPoint { get; set; }
+
+        /// <summary>
+        /// Subscription Id
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Filter on source
+        /// </summary>
+        public Uri SourceFilter { get; set; }
+
+        /// <summary>
+        /// Filter on subject
+        /// </summary>
+        public string SubjectFilter { get; set; }
+
+        /// <summary>
+        /// Filter on alternative subject
+        /// </summary>
+        public string AlternativeSubjectFilter { get; set; }
+
+        /// <summary>
+        /// Filter for type. Starts with
```

Comment:
I think this summary could be a bit more clear

Reply:
updated